protinfo_linux.go

```diff
@@ -19,38 +20,38 @@ func LinkGetProtinfo(link Link) (Protinfo, error) {
 		return pi, err
 	}
 
-	for _, m := range msgs {
-		ans := nl.DeserializeIfInfomsg(m)
+	for i := range msgs {
```

Comment:
msgs has reference type anyway, there is no copy here
and you should avoid premature optimizations, sacrificing readability